.travis.yml

```diff
@@ -1,20 +1,26 @@
 language: ruby
+dist: xenial
 cache: bundler
 
 rvm:
   - 2.3.8
-  - 2.4.5
-  - 2.5.3
+  - 2.4.6
+  - 2.5.5
+  - 2.6.2
 
 matrix:
   exclude:
-    - rvm: 2.4.5
+    - rvm: 2.4.6
       gemfile: gemfiles/rails_40.gemfile
-    - rvm: 2.4.5
+    - rvm: 2.4.6
       gemfile: gemfiles/rails_41.gemfile
-    - rvm: 2.5.3
+    - rvm: 2.5.5
       gemfile: gemfiles/rails_40.gemfile
-    - rvm: 2.5.3
+    - rvm: 2.5.5
+      gemfile: gemfiles/rails_41.gemfile
+    - rvm: 2.6.2
+      gemfile: gemfiles/rails_40.gemfile
+    - rvm: 2.6.2
       gemfile: gemfiles/rails_41.gemfile
 
 gemfile:
```

Comment:
not in the diff but how about dropping rails 3.x?

Reply:
dropped in https://github.com/JordiPolo/kapnismology/pull/15/commits/02e128106c7eae68d44b3f8506bc014cfe9dac27